pom.xml

```diff
@@ -41,5 +41,57 @@
       <type>pom</type>
     </dependency>
   </dependencies>
+  
+  <repositories>
+
+    <!-- releases -->
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>jcenter</id>
+      <name>JCenter Repository</name>
+      <url>https://jcenter.bintray.com/</url>
+    </repository>
+
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>openhab-artifactory-release</id>
+      <name>JFrog Artifactory Repository</name>
+      <url>https://openhab.jfrog.io/openhab/libs-release</url>
+    </repository>
+
+    <!-- snapshots -->
+    <repository>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+        <updatePolicy>always</updatePolicy>
+      </snapshots>
+      <id>openhab-artifactory-snapshot</id>
+      <name>JFrog Artifactory Repository</name>
+      <url>https://openhab.jfrog.io/openhab/libs-snapshot</url>
+    </repository>
+
+    <!-- openHAB dependencies p2 repository -->
+    <repository>
+      <id>p2-openhab-deps-repo</id>
+      <url>https://dl.bintray.com/openhab/p2/openhab-deps-repo/${ohdr.version}</url>
```

Comment:
@kaikreuzer / @maggu2810 this one is now known here, do I need this dependency here and if so how do I define this value just repeat it again in the properties or are there better suggestions?

Reply:
With this I refer to the `${ohdr.version}`

Reply:
As you are using a non Tycho based setup, you should not need any p2 repository anymore.

Reply:
Does that also mean removing the repo subproject, it al looks very p2-like :-)

Reply:
The p2 repo is for consumer of the bundles.
Perhaps we still want to provide p2 repos to keep the option to consume that artifacts in Tycho / PDE based setups / development environments.

Reply:
> Perhaps we still want to provide p2 repos to keep the option to consume that artifacts in Tycho / PDE based setups / development environments.

@kaikreuzer wdyt?

Reply:
I am pretty sure we still want: https://github.com/openhab/openhab-distro/pull/885#issuecomment-466818010